server/cmd/mmctl/commands/post.go

```diff
@@ -100,13 +106,34 @@ func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	channel := getChannelFromChannelArg(c, args[0])
-	if channel == nil {
-		return errors.New("Unable to find channel '" + args[0] + "'")
+	channelID := ""
+	if strings.HasPrefix(args[0], "@") {
```

Comment:
Nit: assign `args[0]` one to a variable and reuse it

Reply:
Okay